### PR TITLE
Update ezeep.rb to latest

### LINKED ISSUE #28401: Cask ezeep.rb needs modification
The Cask for [`ezeep.rb`](https://github.com/caskroom/homebrew-cask/blob/master/Casks/ezeep.rb) currently fails as the `url` and `app` are no longer valid. The `url` is now `https://www.ezeep.com/wp-content/uploads/packages/osx/stable/ezeepInstaller.dmg` which downloads a `.dmg` containing an `.app` called `ezeep Installer.app`.

Running this `.app` calls a GUI requiring a user's password, which downloads _something_ and then installs an `.app` called `ezeep.app`. There is an `uninstall.sh` included with the `ezeep Installer.app` which is a little weird, in particular it requires `sudo` and to be called with a parameter `debug` or `production`, with no explanation given as to what these parameters actually mean. 

Even stranger, there is no similar uninstall script included with the `ezeep.app` that is actually installed.

I am at a loss as to what to do with this Cask, and need some advice please.

### PR DESCRIPTION
**Pull request by miccal**
After making all changes to the cask:

- [X] `brew cask audit --download {{cask_file}}` is error-free.
- [X] `brew cask style --fix {{cask_file}}` reports no offenses.
- [X] The commit message includes the cask’s name and version.

Additionally, 

- [X] `brew cask install {{cask_file}}` worked successfully.
- [X] `brew cask uninstall {{cask_file}}` worked successfully.

Closes https://github.com/caskroom/homebrew-cask/issues/28401

### CODE REVIEW

**Comment by miccal** [comment]
Thanks @vitorgalvao - the formatting of this is very inconsistent among Casks.

File: Casks/ezeep.rb [hunk 1]
```diff
@@ -2,9 +2,15 @@
...
+  url 'https://www.ezeep.com/wp-content/uploads/packages/osx/stable/ezeepInstaller.dmg'
   name 'ezeep'
   homepage 'https://www.ezeep.com/'
 
-  app 'ezeep.app'
+  installer manual: 'ezeep Installer.app'
+
+  uninstall script: {
+                      executable: "#{staged_path}/ezeep Installer.app/Contents/Resources/uninstall.sh",
+                      args:       ['production'],
```
Comment (vitorgalvao) [inline Casks/ezeep.rb]: Does this need to be an array? Does it not work as a string?

Comment (miccal) [inline Casks/ezeep.rb]: I honestly do not know - see for example [zendserver.rb](https://github.com/caskroom/homebrew-cask/blob/master/Casks/zendserver.rb) and [intel-haxm.rb](https://github.com/caskroom/homebrew-cask/blob/master/Casks/intel-haxm.rb), which do the same thing.

Comment (miccal) [inline Casks/ezeep.rb]: @reitermarkus can you enlighten us on this?

Comment (reitermarkus) [inline Casks/ezeep.rb]: This is passed to `SystemCommand`, which only takes an array.

**Review by vitorgalvao — APPROVED** [review]

**Pull request merged**